core/loadpoint_phases.go

```diff
@@ -10,13 +10,16 @@ func (lp *Loadpoint) setConfiguredPhases(phases int) {
 	lp.Lock()
 	defer lp.Unlock()
 
-	lp.ConfiguredPhases = phases
+	lp.configuredPhases = phases
 
 	// publish 1p3p capability and phase configuration
 	if _, ok := lp.charger.(api.PhaseSwitcher); ok {
-		lp.publish(keys.PhasesConfigured, lp.ConfiguredPhases)
+		lp.publish(keys.PhasesConfigured, lp.configuredPhases)
+		lp.settings.SetInt(keys.PhasesConfigured, int64(lp.phases))
 	} else {
+		// TODO why are we publishing nil? Should be 3 according to NewLoadpointFromConfig?
```

Comment:
Aktuell ist es so, dass die UI die Phaseneinstellung nur dann anzeigt, wenn der Charger auch 1p3p unterstützt. Heißt das war auch der einzige Anwendungsfall, wo die UI die konfigurierte Phasenanzahl wissen musste.

Wir sollten das jetzt entkoppeln und explizit die Fähigkeit der Phasenumschaltung als separaten Wert `phaseSwitchingSupported: true` (o.ä.) publishen. Dann entspricht `configuredPhases` in der UI auch immer dem Wert in der Konfiguration.
Unterstützt der LP Phasenumschaltung gibts neben 1/2/3 in der UI auch noch die Möglichkeit "auto" zu wählen.